common/config/config.json.template

```diff
@@ -23,8 +23,29 @@
   "logFileName" : "~/dvr/kLiveController.log",
   "logTimeZoneOffset" : 300,
   "monitorPort": 4390,
+  "preserveOriginalHLS": false,
+  "recordingDurationMinutes": 5,
```

Comment:
can you please move that to regessionAdapter or such?  it will be confusing for the future when we will do the real live to vod recording in the live controller